File: 0122-best-time-to-buy-and-sell-stock-ii/0122-best-time-to-buy-and-sell-stock-ii.py
class Solution(object):
    def maxProfit(self, prices):
        """
        :type prices: List[int]
        :rtype: int
        """
        max=0
        start=prices[0]
        n=len(prices)
        for i in range(0,n):
            if start<prices[i]: 
                max+=prices[i]-start
            start=prices[i]
        return max

File: 0122-best-time-to-buy-and-sell-stock-ii/0122-best-time-to-buy-and-sell-stock-ii.js
/**
 * @param {number[]} prices
 * @return {number}
 */
var maxProfit = function(prices) {
    let start=prices[0],m=0;
        for(let i=1;i<prices.length;i++){
            if(start<prices[i]) m+=prices[i]-start;
            start=prices[i];
        }
        return m;
};